webcompat/static/js/lib/wizard/steps/upload-helper/upload-text.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import notify from "../../notify.js";
+
+const changeUploadText = textId => {
```

Comment:
I guess the newest prettier is going to complain about this right? Maybe good to rebase against master and re-run prettier?